src/test/java/UITests/LoginSteps.java

```diff
@@ -39,28 +46,14 @@ public void page_should_be_refreshed() {
         throw new io.cucumber.java.PendingException();
     }
 
-    @Given("I enter numbers on the username")
-    public void i_enter_numbers_on_the_username() {
-        // Write code here that turns the phrase above into concrete actions
-        throw new io.cucumber.java.PendingException();
-    }
-
-    @Given("I let password as white space")
-    public void i_let_password_as_white_space() {
-        // Write code here that turns the phrase above into concrete actions
-        throw new io.cucumber.java.PendingException();
-    }
 
-    @When("I press the login button")
-    public void i_press_the_login_button() {
-        // Write code here that turns the phrase above into concrete actions
-        throw new io.cucumber.java.PendingException();
-    }
 
     @Then("login shouldn't work")
     public void login_shouldn_t_work() {
-        // Write code here that turns the phrase above into concrete actions
-        throw new io.cucumber.java.PendingException();
+        WebElement errortext= driver.findElement(By.xpath("/html/body/div/div/div[2]/div[1]/div[1]/div/form/div[3]/h3"));
```

Comment:
de evitat acest stil, incearca sa gasesti altfel sau sa folosesti expresii regulate, altfel daca apare un element in viitor in acea cale - orice buton orice input nu-ti va mai functiona testele

Reply:
ok